DAGs.md

```diff
@@ -536,56 +534,6 @@ Notes:                  https://stocksnap.io/api/load-photos/date/desc/1
                         API is undocumented.
 
 
-## `tsv_to_postgres_loader`
-
-#### Database Loader DAG
-**DB Loader Apache Airflow DAG** (directed acyclic graph) takes the media data saved
-locally in TSV files, cleans it using an intermediate database table, and saves
```

Comment:
I was about to ask if we don't need to conserve this documentation but I see we have it where it was moved :)
https://github.com/WordPress/openverse-catalog/blob/3217ed5ecf3f9babeb96929e8883110e5da1c199/openverse_catalog/dags/database/loader_workflow.py#L2-L6

Reply:
I made sure to check that as well 😄 Should have made a note in the description, apologies!